models/migrations/20191004135959_create-free-token-column.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function(knex) {
+  return knex.schema.alterTable('users', (tbl) => {
+    tbl.bigInteger('credits').notNullable().defaultTo(1);
```

Comment:
This can just be `tbl.integer` since it should be a fairly small number.  Big integer covers really large numbers